### DISCUSSION TITLE
Unable to build the InStore Code using the Pipeline

### DISCUSSION THREAD

**Discussion by Gopikrish25** [microsoft/Dynamics365Commerce.Solutions#23] [Q&A] [answered]
Hello,

I was trying to build the Standard 9.39 version of the InStore code using the YAML file reference and the build solution step is getting errored out with the attached errors.

Please let me know how to resolve this error.

Thanks.

Attachments:
[logs_114.zip](https://github.com/microsoft/Dynamics365Commerce.Solutions/files/10243187/logs_114.zip)
[Errors.txt](https://github.com/microsoft/Dynamics365Commerce.Solutions/files/10243189/Errors.txt)

**Comment by madyke** [comment, marked as answer]
FYI @lukedgr to investigate updating/removing the MPOS build portion of the pipeline, when he's available.

**Reply by Gopikrish25** [reply]
Thank you @madyke
